independent-publisher-connectors/Financial Industry Regulatory Authority FINRA (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,329 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "FINRA Custom Connector for Power Platform",
```

Comment:
Please update the title according to the link below. Please remove "Power Platform" and ensure that the title should be within 30 characters. 

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title